content/components/pagehead.mdx

```diff
@@ -42,6 +42,17 @@ Pagehead is a heading component for the title and start of a given page section.
 
 ## Accessibility
 
+### Heading usage
+Pagehead does not provide a heading by default. Since this component is intended to be used for the title and start of a given page section, it should include a heading (`<h1>-<h6>`) in most cases. 
```

Comment:
```suggestion
`Pagehead` does not provide a heading by default. Since this component is intended to be used for the title and start of a given page section, it should include a heading (`<h1>-<h6>`) in most cases. 
```